main.py

```diff
@@ -49,62 +49,58 @@ def home():
 
 @app.route("/v1/spec")
 def spec():
-    swag = swagger(app)
+    swag = swagger(app, from_file_keyword="swagger_from_file")
     swag['info']['version'] = "1.0"
     swag['info']['title'] = "Sunnah.com API"
```

Comment:
Would be good to add host property `"host": "api.sunnah.com"` as that would allow directly calling the API from swagger UI.